data_analysis/plots.py

```diff
@@ -678,172 +818,392 @@ def run_mvp() -> None:
+    daily_means = calculate_trips_per_rider(merged_df)
 
-    # Quantile plot
-    logger.info("Creating map of all routes binned by quintile")
-    bounds = legend_formatter(summary_gdf_geo, "ratio", decimals="2f")
+    summary_df_mean = summary_df.merge(daily_means, on='route_id')
 
-    summary_kwargs_quantiles = {
-        "cmap": "plasma",
-        "column": "ratio",
-        "scheme": "Quantiles",
-        "legend_kwds": {
-            'caption': "Ratio of Actual Trips to Scheduled Trips",
-            'colorbar': False,
-            'labels': bounds
-        },
-        "legend": True,
-        "categorical": False,
-        "k": 5
-    }
-    save_name = f"all_routes_quantiles_{start_date}_to_{end_date}_all_day_types"
+    summary_df_mean = calculate_percentile_and_rank(
+        summary_df_mean,
+        col="avg_trip_count_rt"
+    )
 
-    make_ratio_map(
-        summary_gdf_geo=summary_gdf_geo,
-        save_name=save_name,
-        summary_kwargs=summary_kwargs_quantiles
+    summary_df_mean = calculate_percentile_and_rank(
+        summary_df_mean,
+        col="trips_per_1000_riders"
     )
 
-    # Worst performing routes
-    logger.info("Creating map of worst performing routes")
+    summary_gdf = summary_df_mean.merge(gdf, how="right", on="route_id")
 
-    worst_geo = n_worst_best_routes(summary_gdf_geo, percentile=False)
+    summary_gdf_geo = gpd.GeoDataFrame(summary_gdf)
 
-    save_name = f"worst_routes_{start_date}_to_{end_date}_all_day_types"
+    make_all_maps(
+        summary_gdf_geo=summary_gdf_geo,
+        start_date=start_date,
+        end_date=end_date
+    )
 
-    summary_kwargs['legend_kwds'] = {
-        "caption": "Ratio of Actual Trips to Scheduled Trips"
-    }
-    summary_kwargs['cmap'] = 'winter'
+    create_ridership_map()
 
-    make_ratio_map(
-        summary_gdf_geo=worst_geo,
-        save_name=save_name,
-        summary_kwargs=summary_kwargs
+    make_descriptive_plots(
+        combined_long_df=combined_long_df,
+        summary_df=summary_df
     )
 
-    worst_geo_path = create_save_path(save_name, DATA_PATH)
-    worst_geo.to_file(f'{worst_geo_path}.json', driver="GeoJSON")
-    worst_geo.to_html(f'{worst_geo_path}_table', index=False)
 
-    # Best routes
-    logger.info("Creating map of best performing routes")
-
-    best_geo = n_worst_best_routes(
-        summary_gdf_geo,
-        percentile=False,
-        worst=False
-    )
-    save_name = f"best_routes_{start_date}_to_{end_date}_all_day_types"
+def merge_ridership_combined(
+    combined_long_df: pd.DataFrame,
+    ridership_df: pd.DataFrame,
+    start_date: str,
+        ridership_end_date: str) -> pd.DataFrame:
+    """Merge the combined_long_df and ridership data
 
-    summary_kwargs['legend_kwds'] = {
-        "caption": "Ratio of Actual Trips to Scheduled Trips"
-    }
+    Args:
+        combined_long_df (pd.DataFrame): The first element of the output from
```

Comment:
Can we expand a bit on what this dataframe actually contains here? This is helpful to explain where it is generated but might be nice to summarize its actual contents (like, what columns is it expected to have.)

Reply:
Sure, I can add the head of the DataFrame to the docstrings so that it's more clear what it looks like.